common/common.sh

```diff
@@ -359,7 +359,7 @@ dmesg_pattern_check() {
 # Record last timestamp in dmesg and store the value in variable
 # LAST_DMESG_TIMESTAMP. The value is refered in function extract_case_dmesg.
 last_dmesg_timestamp() {
-  LAST_DMESG_TIMESTAMP=$(dmesg | tail -n1 | awk '{print $1}' | tr -d "[]")
+  LAST_DMESG_TIMESTAMP=$(dmesg | tail -n1 | awk -F "]" '{print $1}' | tr -d "[]")
```

Comment:
@xupengfe , can you help explain why we need to set "]" as field seperator explicitly?

Reply:
If there is no  -F "]",  awk will check the first space as  "field seperator"


If there is space in last line dmesg time stamp, it will got null time stamp, it's wrong.
Like below example,  if there is no  -F "]",  will get null time stamp.
[  275.926781] xxxx

Thanks!

Reply:
Got it. Thanks.